definitions/procedures/restore/regenerate_queues.rb

```diff
@@ -0,0 +1,63 @@
+module Procedures::Restore
+  class RegenerateQueues < ForemanMaintain::Procedure
```

Comment:
Is this procedure valid only when `pulp2` feature enabled?

Reply:
I think this should be for pulp2 only as I can see we dont have qpidd service included in pulpcore. @johnpmitsch am I correct ?

Reply:
yes that is fine to gate this functionality on pulp2. Pulp 2 will still be present for a while, but it will be helpful when we do drop pulp2.